roles/studio_proc/molecule/default/verify.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Verify
+  hosts: all
+  gather_facts: false
+  tasks:
+  - name: test for mount directories
+    stat:
+      path: "/mnt/{{ item }}"
+    register: smb_dir
+    loop:
+      - libimages
+      - diglibdata
+      - hydra_sources
+    failed_when:
+      - smb_dir.stat.isdir is not defined
```

Comment:
Will this test fail if only two of the three Samba directories get mounted correctly?